ticdc/ticdc-alert-rules.md

```diff
@@ -39,6 +39,20 @@ summary: 了解 TiCDC 集群监控报警规则以及处理方法。
 
     该告警与同步任务中断类似，可参考 [TiCDC 同步任务出现中断](/ticdc/troubleshoot-ticdc.md#ticdc-同步任务出现中断)的处理方法。
 
+### `ticdc_changefeed_failed`
+
+* 报警规则：
+
+    `(max_over_time(ticdc_owner_status[1m]) == 2) > 0`
+
+* 规则描述：
+
+    TiCDC 某个同步任务遇到无法自动恢复的错误，进入 Failed 状态。
```

Comment:
```suggestion
    TiCDC 某个同步任务遇到无法自动恢复的错误，进入 failed 状态。
```
Keep it consistent with: https://github.com/pingcap/docs-cn/blob/a56b50776d9c1f416ec7b9a5b0755e53b284d1ae/ticdc/ticdc-open-api.md?plain=1#L253-L257